RELEASING.md

```diff
@@ -0,0 +1,12 @@
+# Releasing
```

Comment:
There is no support for the multi-platform Docker images on DockerHub but they have it on the roadmap: https://github.com/docker/roadmap/issues/109.

Reply:
@rombert GH Actions supports `buildx` well. You may push images from GH Actions temporarily.

Reply:
@tomaszmichalak - the main issue here is that we don't have access to the docker repository credentials, so the `docker login`/`docker push` workflow does not work for us.

We set up automated builds on Docker Hub because that's what works for us with a relatively small effort. If we want to switch to GitHub actions, we need to figure out how that would work, and if ASF infra will provide the necessary secrets for us.

I've started a conversation on the ASF builds mailing to see how we can best approach this.

Reply:
@tomaszmichalak - in the meantime, I started educating myself about `docker buildx` and multi-arch images. I ran into some failures, maybe you have an idea about that?

```
$ docker buildx create --use
$ docker buildx build --platform linux/arm64/v8,linux/amd64 --tag apache/sling:snapshot-multi .
```

This eventually fails with

```
 => ERROR [linux/arm64 6/7] RUN find /opt/sling/artifacts -type f -perm 0600 | xargs --no-run-if-empty chmod 0644                                                                                               0.4s 
------                                                                                                                                                                                                               
 > [linux/arm64 6/7] RUN find /opt/sling/artifacts -type f -perm 0600 | xargs --no-run-if-empty chmod 0644:                                                                                                          
#0 0.311 Error while loading /usr/local/openjdk-17/bin/chmod: No such file or directory
------
Dockerfile:40
--------------------
  38 |     # ensure all files are readable by the sling user
  39 |     # for some reason some jar files are 0600 while most are 0644
  40 | >>> RUN find /opt/sling/artifacts -type f -perm 0600 | xargs --no-run-if-empty chmod 0644
  41 |     
  42 |     USER sling:sling
--------------------
ERROR: failed to solve: process "/dev/.buildkit_qemu_emulator /bin/sh -c find /opt/sling/artifacts -type f -perm 0600 | xargs --no-run-if-empty chmod 0644" did not complete successfully: exit code: 12
```

Does the build work for you?

Reply:
My steps:

1. Check existing BuildKits:
```bash
> docker buildx ls
NAME/NODE       DRIVER/ENDPOINT STATUS  BUILDKIT PLATFORMS
default *       docker
  default       default         running 20.10.21 linux/arm64, linux/amd64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6
desktop-linux   docker
  desktop-linux desktop-linux   running 20.10.21 linux/arm64, linux/amd64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6
```
2. Create the new BuildKit configuration:
```bash
> docker buildx create --use
pedantic_shockley
```
3. Build project to init all files in `target`:
```bash
mvn package
```
4. Build images with `--load` (`--output=type=docker`) option
```bash
> docker buildx build --platform linux/arm64  -t tomaszmichalaka/sling-starter:20221208 . --output=type=docker
...
 => CACHED [linux/arm64 5/7] COPY target/artifacts/ /opt/sling/artifacts/                                                                                                                                                                        0.0s
 => CACHED [linux/arm64 6/7] RUN find /opt/sling/artifacts -type f -perm 0600 | xargs --no-run-if-empty chmod 0644                                                                                                                               0.0s
 => CACHED [linux/arm64 7/7] WORKDIR /opt/sling 
```
5. List docker images:
```bash
docker images                                                                                                                                      ✔ ╱ 8s ╱ 09:31:20 
REPOSITORY                                                TAG                                                                          IMAGE ID       CREATED          SIZE
tomaszmichalaka/sling-starter                             20221208                                                                     3a5fb2966493   15 minutes ago   520MB
```

Please note that `docker buildx build --load` supports only one single platform. For multiple platforms, you need to push images to the remote registry.